graph/builder.go

```diff
@@ -313,7 +315,6 @@ func (b *Builder) Stop() error {
 	}
 
 	log.Info("Builder shutting down...")
-	defer log.Debug("Builder shutdown complete")
```

Comment:
so the question is, do we really wanna print shutdown complete if and error happened, I guess not ?